docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -669,9 +669,9 @@ in PEM format.
     ... )
 
 
-.. _`FIPS 186-3`: http://csrc.nist.gov/publications/fips/fips186-3/fips_186-3.pdf
+.. _`FIPS 186-3`: https://csrc.nist.gov/csrc/media/publications/fips/186/3/archive/2009-06-25/documents/fips_186-3.pdf
 .. _`FIPS 186-4`: http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.186-4.pdf
```

Comment:
https://csrc.nist.gov/publications/detail/fips/186/4/final

Reply:
That's on a different domain, I'm doing the nvlpubs ones in a separate PR. (nvlpubs was not affected by this migration)

Reply:
This can be in a  subsequent PR if we want. There are several more non-https NIST URLs still present.